std/net/isemail.d

```diff
@@ -1953,7 +1922,6 @@ ElementType!(A) pop (A) (ref A a) if (isDynamicArray!(A) && !isNarrowString!(A)
     return e;
 }
 
-///
 unittest
 {
     auto array = [0, 1, 2, 3];
```

Comment:
Seems like there is already [shift](https://github.com/dlang/phobos/pull/4010) within Phobos ;-)
(though this one is private)